src/components/Header/index.jsx

```diff
@@ -42,17 +44,31 @@ const Header = () => {
               </Link>
             </li>
           ) : (
-            <li>
-              <button
-                className="btn btn-secondary auth-button"
-                onClick={() => {}}
-              >
-                Logout
-              </button>
-            </li>
+            <>
```

Comment:
You can remove the fragment tag since li tag is already a container here

Reply:
I will do that